test/dnstest_SUITE.erl

```diff
@@ -91,6 +101,9 @@ assert(_Definitions, Results) ->
 %% until we can run all of them
 failing() ->
     [
+        nsec_alias_a,
+        nsec_alias_aaaa,
+        nsec_alias_other,
```

Comment:
This makes me think, @DXTimer, don't we want to maybe put these into a different definitions section in dnstest? As in https://github.com/dnsimple/dnstest/blob/05e81877e8e36058eea0932951b64eb04c7be27a/src/dnstest_definitions.erl#L5-L7, perhaps a `erldnsimple_definitions/0` helper? So that we don't need to ignore them here but may just not run them.

Reply:
We should not have these failing. We want to have a 100% coverage so having tests that we expect to fail should not be a common scenario but an outlier in between feature branches, etc.

Reply:
but this is erldns, not erldnsimple, there's no support for alias records here.